packages/nimble/R/nimbleProject.R

```diff
@@ -508,8 +520,10 @@ nimbleProjectClass <- setRefClass('nimbleProjectClass',
                                      modelDef <- model$getModelDef()
                                      modelDefName <- modelDef$name
                                      Cname <- Rname2CppName(modelDefName)
-                                     if(is.null(filename)) filename <- paste0(projectName, '_', Rname2CppName(modelDefName)) 
-                                     
+                                     if(is.null(filename)) {
+                                         filename <- paste0(projectName, '_', Rname2CppName(modelDefName)) 
+                                         nfFileName <- paste0(projectName, '_', Rname2CppName(modelDefName),'_nfCode')
```

Comment:
Should `nfFileName` happen outside of this `if` statement? (I'm just confirming that you intended the behavior change.)